transaction_record.go

```diff
@@ -77,6 +78,11 @@ func _TransactionRecordFromProtobuf(pb *services.TransactionRecord) TransactionR
 		tokenAssociation = append(tokenAssociation, tokenAssociationFromProtobuf(association))
 	}
 
+	var alias *[]byte
+	if len(pb.Alias) != 0 {
```

Comment:
Is this the correct comparison? I assume `Alias` can be `nil`

Reply:
No in go arrays are not nil just empty, and they don't have it as a pointer in protobuf